go.mod

```diff
@@ -4,18 +4,18 @@ go 1.17
 
 require (
 	github.com/Jeffail/gabs/v2 v2.6.0
-	github.com/ThreeDotsLabs/watermill v1.1.1
+	github.com/ThreeDotsLabs/watermill v1.3.2
 	github.com/caarlos0/env/v6 v6.10.1
-	github.com/cenkalti/backoff/v3 v3.0.0
+	github.com/cenkalti/backoff/v3 v3.2.2
```

Comment:
The cenkalti/backoff/v3 version has increased also, but it is not updated in the NOTICE file. Use _go version -m <suite-connector-binary>_ to list the versions of the build dependencies that goes in the binary and update them in the NOTICE file.